.svglintrc.js

```diff
@@ -473,6 +473,40 @@ module.exports = {
                 ignoreIcon(reporter.name, iconPath, $);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = "path-format";
+
+            const iconPath = $.find("path").attr("d");
+
+            const validPathFormatRegex = /^[Mm][MmZzLlHhVvCcSsQqTtAaEe0-9-,.\s]+$/;
+            if (!validPathFormatRegex.test(iconPath)) {
+              let errorMsg = "Invalid path format", reason;
+
+              if (!(/^[Mm]/.test(iconPath))) {
+                // doesn't start with moveto
+                reason = `should start with \"moveto\" command (\"M\" or \"m\"), but starts with \"${iconPath[0]}\"`;
+                reporter.error(`${errorMsg}: ${reason}`);
+              }
+
+              const validPathCharacters = "MmZzLlHhVvCcSsQqTtAaEe0123456789-,. ",
+                    invalidCharactersMsgs = [],
+                    pathDStart = '<path d="',
+                    pathDIndex = $.html().indexOf(pathDStart) + pathDStart.length;
+
+              for (let i=1; i<iconPath.length; i++) {
+                if (validPathCharacters.indexOf(iconPath[i]) === -1) {
+                  invalidCharactersMsgs.push(`"${iconPath[i]}" at index ${pathDIndex + i}`);
+                }
+              }
```

Comment:
This seems more complicated that it needs to be, can't we just loop over the characters in `iconPath`with a `for-in`/`for-of` loop? Or define the `validPathCharacters` as a "negative" regular expression (i.e. `/[^...]/`) and use [`RegExp.exec`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec) instead?

The latter may be too complicated for this script, so I would be fine with either.

Reply:
Something like [this](https://github.com/simple-icons/simple-icons/pull/5069/commits/a69cedef897452ab1ac5f17d10edcf837ca5175b#diff-6a7a61d95c2af287ca7aae4861a6720e553f5aaebcfab1e90b1189bd6639f14cR497-R499)?